src/main/java/ro/utcn/sp1tz3/Assignment1/controller/ConsoleController.java

```diff
@@ -0,0 +1,254 @@
+package ro.utcn.sp1tz3.Assignment1.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import ro.utcn.sp1tz3.Assignment1.entity.Answer;
+import ro.utcn.sp1tz3.Assignment1.entity.Question;
+import ro.utcn.sp1tz3.Assignment1.entity.Tag;
+import ro.utcn.sp1tz3.Assignment1.entity.User;
+import ro.utcn.sp1tz3.Assignment1.exception.UserNotFoundException;
+import ro.utcn.sp1tz3.Assignment1.repository.TagRepository;
+import ro.utcn.sp1tz3.Assignment1.service.AnswerManagementsService;
+import ro.utcn.sp1tz3.Assignment1.service.QuestionManagementService;
+import ro.utcn.sp1tz3.Assignment1.service.TagManagementService;
+import ro.utcn.sp1tz3.Assignment1.service.UserManagementService;
+
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Date;
```

Comment:
I would strongly recommend against using java.util.Date, java.sql.Time, etc. Instead check out the newer Java 8 time classes (LocalDate, LocalTime, etc).